### ISSUE TITLE
ColorPicker More/Less button has incorrect a11y help text

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/microsoft-ui-xaml#7395]
### Describe the bug

See https://github.com/microsoft/terminal/issues/12046

The help text for the More/Less button is currently: "Invoke to show or hide the text entry fields."

When "More" is shown, it should be "Invoke to show the text entry fields.".
When "Less" is shown, it should be "Invoke to hide the text entry fields.".

Basically, it should be more specific.

### Steps to reproduce the bug

1. In WinUI2 Gallery, go to: [winui2gallery://item/ColorPicker](winui2gallery://item/ColorPicker)
2. Ensure `IsMoreButtonVisible` is enabled
3. (While using a screen reader), tab to the "More" or "Less" button

### Expected behavior

For 'More' button Screen Reader should announce 'More button, invoke to show the text entry fields' and for 'Less' button Screen Reader should announce 'Less button, invoke to hide the text entry fields'

### Windows app type

- [X] UWP
- [ ] Win32

### Device form factor

Desktop

### Additional context

WinUI 2 Gallery version: 1.2.25.0

**Comment by zadjii-msft** [comment]
(Internal bug: MSFT:40517763)

**Comment by zadjii-msft** [comment]
This is still an issue.
